shared/loadgenerator/loadgenerator.go

```diff
@@ -37,12 +37,14 @@ const (
 	p99      = 99.0
 	duration = 1 * time.Minute
 	qps      = 10
+	numSteps = 1
 	jsonExt  = ".json"
 )
 
 // GeneratorOptions provides knobs to run the perf test
 type GeneratorOptions struct {
 	// Duration is the total time to generate the load.
+	// If NumSteps is specified, this value will hold true for each step.
```

Comment:
Do you mean if NumSteps is specified, this value is the duration of each step?  How about "Duration is the time to generate load in each step."?